Source/GTMSessionFetcher.m

```diff
@@ -18,6 +18,11 @@
 #endif
 
 #import "GTMSessionFetcher.h"
+#if TARGET_OS_OSX && GTMSESSION_RECONNECT_BACKGROUND_SESSIONS_ON_LAUNCH
```

Comment:
Actually, the `#define` is initialized below, so this isn't completely what you want.  Either it moves back to the .h or you likely need to move this down until after you default it.

Reply:
I moved the default define to the header file, since it's essentially part of the public API.